validator/sawtooth_validator/server/cli.py

```diff
@@ -15,7 +15,7 @@
 
 import logging
 import sys
-import argparse
+import json
```

Comment:
Suggest we not use JSON for any serialization between components. While it may be easy, it is not good in that it does create a contract between them. Protobuf would be better.  Avoiding serialization completely would be idea.